src/libfuncs/enum.rs

```diff
@@ -160,39 +133,9 @@ pub fn build_init<'ctx, 'this>(
                     ))
                     .result(0)?
                     .into();
-                let stack_ptr = helper
-                    .init_block()
-                    .append_operation(llvm::alloca(
-                        context,
-                        k1,
-                        llvm::r#type::opaque_pointer(context),
-                        location,
-                        AllocaOptions::new()
-                            .align(Some(IntegerAttribute::new(
-                                IntegerType::new(context, 64).into(),
-                                layout.align() as i64,
-                            )))
-                            .elem_type(Some(TypeAttribute::new(type_info.build(
-                                context,
-                                helper,
-                                registry,
-                                metadata,
-                                &info.branch_signatures()[0].vars[0].ty,
-                            )?))),
-                    ))
-                    .result(0)?
-                    .into();
+            let stack_ptr = helper.alloca(context, location, k1, Some(layout.align()))?;
```

Comment:
look below

Reply:
oh, adding the operations to init block?

Reply:
Yes, please. Otherwise the code is not equivalent to the original.

Reply:
right! fixed.